src/store/metadata/oxigraph/store.py

```diff
@@ -110,7 +118,60 @@ def get_edition(
         """
         Gets a specific edition of a specific dataset
         """
-        raise NotImplementedError
+        # Populate the graph from the database
+        graph = self.db
+
+        # Use the construct wrappers to pull the raw RDF triples
+        # (as one rdflib.Graph() for each function) and add them
+        # together to create a single Graph of the
+        # data we need.
+        result: Graph = (
+            construct_edition_core(graph, dataset_id, edition_id)
+            + construct_edition_contact_point(graph, dataset_id, edition_id)
+            + construct_edition_topics(graph, dataset_id, edition_id)
+            + construct_edition_keywords(graph, dataset_id, edition_id)
+            + construct_edition_temporal_coverage(graph, dataset_id, edition_id)
+            + construct_edition_table_schema(graph, dataset_id, edition_id)
+            + construct_edition_versions(graph, dataset_id, edition_id)
+        )
+
+        # Serialize the graph into jsonld
+        data = json.loads(result.serialize(format="json-ld"))
+
+        # Use a context file to shape our jsonld, removing long form references
+        data = jsonld.flatten(
+            data, {"@context": constants.CONTEXT, "@type": "dcat:Dataset"}
+        )
+
+        edition_graph = _get_single_graph_for_field(data, "@type")
+        contact_point_graph = _get_single_graph_for_field(data, "vcard:fn")
+        temporal_coverage_graph = _get_single_graph_for_field(data, "dcat:endDate")
```

Comment:
```
if None in [edition_graph, contact_point_graph, temporal_coverage_graph]:
         return None
```

its not perfect (there's no guarantee the dicts include the keys we expect them to - though sparql being sparql thats unlikely to be a thing) but it'll guard us against uncontrolled exceptions if we try and reference a key inside a None further down the function